src/Rate.php

```diff
@@ -190,6 +190,16 @@ private function createRequest(RateRequest $rateRequest)
         if (isset($deliveryTimeInformation)) {
             $shipmentNode->appendChild($deliveryTimeInformation->toNode($xml));
         }
+          
+        $ShipmentTotalWeight = $shipment->getShipmentTotalWeight();
+        if (isset($ShipmentTotalWeight)) {
+            $shipmentNode->appendChild($ShipmentTotalWeight->toNode($xml));
+        }
+        
+        $InvoiceLineTotal = $shipment->getInvoiceLineTotal();
```

Comment:
This one seems unrelated to `shipmentTotalWeight`. Is this intended?

Reply:
Hey sorry. I think the title is a little misleading. I was trying use a Shoptimeintrasnsit request to PR and got error 111546 and then 111549, which lead me to this page:

I had to add both unit of measure and shipment total time to get it to work.

https://developerkitcommunity.ups.com/index.php/Special:AWCforum/st/id449474/

Reply:
Ok, but can you take this one out and put it in a separate PR? The thing is that the commit should reflect the change and should preferably be a single change instead of combined changes :)